test/fixtures/keys/Makefile

```diff
@@ -24,37 +24,37 @@ all: \
   dh512.pem \
   dh1024.pem \
   dh2048.pem \
-	dherror.pem \
```

Comment:
thanks! the whitespace was broken. Can you add a commit message description saying what the change is: "Make the indent whitespace consistent."

Reply:
Added!